compilers/concrete-compiler/compiler/lib/Support/V0Parameters.cpp

```diff
@@ -287,7 +287,9 @@ llvm::Error checkPErrorSolution(Solution solution, optimizer::Config config) {
     return StreamStringError() << "Cannot find crypto parameters";
   }
 
-  if (/*descr.dag &&*/ !config.display && /*naive_user &&*/
+  bool naive_config = (std::isnan(config.global_p_error) &&
```

Comment:
global_p_error is always set to a value, it's never `nan` so I'm not sure if this will work as expected :thinking: If it's not explicitly set, it's set to `1.0` so maybe that's what we can check :thinking:

Reply:
On compiler side NAN is the default value for this parameter.

Reply:
But you're right it should be && instead of || . I only tested on compiler not on concrete-python.